server/routes/chat_routes.ts

```diff
@@ -198,6 +198,19 @@ export function registerChatRoutes(router: IRouter, routeOptions: RoutesOptions)
         return response.custom({ statusCode: error.statusCode || 500, body: error.message });
       }
 
+      if (outputs.stream) {
+        const result = response.ok({
+          headers: {
+            'Content-Encoding': 'identity',
```

Comment:
Minor: should we add some explanation for the reason of this?

Reply:
Sure, I added some comment in the fetch module but we should add comment in here as well.